src/lib/q/query.go

```diff
@@ -29,6 +29,20 @@ type Query struct {
 	Sorting string
 }
 
+// GetString get value of key in Keywords and convert it to string
+func (q *Query) GetString(key string) string {
```

Comment:
what if the empty string meet the query condition？

Reply:
removed as it is not used